tests/unittests/tests-ztimer/tests-ztimer-convert-muldiv64.c

```diff
@@ -120,7 +120,7 @@ static void test_ztimer_convert_muldiv64_set_speedup(void)
     TEST_ASSERT_EQUAL_INT(4294968LU, ztimer_now(&zmock.super));
 
     /* convert has overflowed ((2000 + 4294966000) % 2**32 = 704U)*/
-    TEST_ASSERT_EQUAL_INT(704LU, ztimer_now(z));
+    TEST_ASSERT_EQUAL_INT(704LU, (uint32_t) ztimer_now(z));
```

Comment:
``/*cast in case ztimer_now_t is uin64_t*/`
would this help?`

Reply:
yup!